projects/aca-content/src/lib/components/search/search-save/dialog/delete/saved-search-delete-dialog.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="aca-saved-search-delete-dialog__header">
+  <h2 class="aca-saved-search-delete-dialog__title">{{"APP.BROWSE.SEARCH.SAVE_SEARCH.DELETE_DIALOG.TITLE" | translate}}</h2>
+  <button mat-icon-button mat-dialog-close><mat-icon>close</mat-icon></button>
+</div>
+<mat-dialog-content>
+<p>{{ 'APP.BROWSE.SEARCH.SAVE_SEARCH.DELETE_DIALOG.CONTENT' | translate }}</p>
+</mat-dialog-content>
+<mat-dialog-actions align="end">
+  <button mat-button
+          mat-dialog-close
+          id="aca-save-search-delete-dialog-cancel-button">{{ 'CANCEL' | uppercase | translate }}</button>
```

Comment:
Is this supposed to be uppercase or titlecase? As per the designs on the JIRA, it looks to be titlecase